presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix incorrect query result for queries involving timestamp, when the time zone database
+  embedded in the version of Joda-Time that Presto depends on does not match the time zone
+  database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when ``optimize_mixed_distinct_aggregation`` is enabled.
```

Comment:
We need double backticks around `count(*)` as `*` will make the rest of the sentence italic.

```
``count(*)``
```

Reply:
What does "fix" mean? Did it fail during planning? Return an incorrect result?

Reply:
@sopel39 Karol, could you answer David's question?

Reply:
Maybe:
```
Do not return ``NULL`` as ``count(*)`` aggregation result when ``optimize_mixed_distinct_aggregation`` is enabled.
```
?